geo/src/main/java/org/odk/collect/geo/SelectionMapFragment.kt

```diff
@@ -0,0 +1,416 @@
+package org.odk.collect.geo
```

Comment:
Now in this file we have a few classes  what makes it big are you going to split it?

Reply:
Yeah it feels messy. How about we discuss the package structure as part of the next PR, so it's as closed to "finished" as possible? That way we hopefully avoid any churn on things moving around.